crypto-intrinsics/src/x86_64.rs

```diff
@@ -0,0 +1,62 @@
+//! `x86_64` intrinsics
+
+use core::arch::asm;
+
+/// Move if zero.
+///
+/// Uses a `test` instruction to check if the given `condition` value is
+/// equal to zero, then calls `cmovz` (a.k.a. `cmove`) to conditionally move
+/// `src` to `dst` when `condition` is equal to zero.
+#[inline]
+pub fn cmovz(condition: u64, src: u64, dst: &mut u64) {
```

Comment:
I think you can use `#[inline(always)]` on such intrinsics.

Reply:
Added